src/DIRAC/Core/Utilities/test/Test_entrypoints.py

```diff
@@ -0,0 +1,13 @@
+import importlib_metadata as metadata
+
+
+def test_entrypoints():
+    """Make sure all console_scripts defined by LHCbDIRAC are importable."""
+    errors = []
+    for ep in metadata.entry_points(group="console_scripts"):
+        if ep.module.startswith("LHCbDIRAC"):
```

Comment:
I guess it should `DIRAC` instead of `LHCbDIRAC` here, shouldn't it?
```suggestion
        if ep.module.startswith("DIRAC"):
```

Reply:
🤦